qa/tasks/cephfs/kernel_mount.py

```diff
@@ -64,6 +67,8 @@ def mount(self, mntopts=[], createfs=True, check_status=True, **kwargs):
             if 'permission denied' in stderr.getvalue().lower():
                 pass
 
+        self.cluster_fsid = self.getfattr("./", "ceph.cluster_fsid")
+        self.client_gid = self.getfattr("./", "ceph.client_id")
```

Comment:
`get_global_id` can now return this.

Reply:
Sorry, didn't notice it. Checked it, it is buggy.

The logic will is:

The `vstart_runner.py` will set the kernel mountpoint's `self.client_id` to a index from the range of `[0, max_required_clients], and then the `kernel_mount` will list all the `/sys/kernel/debug/ceph/*` dirs in an arrary, and we cannot be sure the dir in the arrary's order will match the `self.client_id`, and it will only always work well if the `max_required_clients` is 1.

Reply:
I will fix it.

Reply:
> Sorry, didn't notice it. Checked it, it is buggy.
> 
> The logic will is:
> 
> The `vstart_runner.py` will set the kernel mountpoint's `self.client_id` to a index from the range of `[0, max_required_clients]`, and then the `kernel_mount` will list all the `/sys/kernel/debug/ceph/*` dirs in an arrary, and we cannot be sure the dir in the arrary's order will match the `self.client_id`, and it will only always work well if the `max_required_clients` is 1.

My bad, I misread the code, this should work.